apps/nfid-frontend/src/features/security/passkey/add-passkey.tsx

```diff
@@ -37,7 +38,14 @@ export const AddPasskey = ({
           })
           toaster.success("Device has been added")
         } catch (e) {
-          toaster.error((e as Error).message)
+          if (e instanceof Error) {
+            toaster.error(
+              ERROR_DEVICE_IN_EXCLUDED_CREDENTIAL_LIST.includes(e.message)
+                ? "This device is already registered"
+                : e.message,
+            )
+          }
+          throw e
```

Comment:
do we need to throw an error here?
Showing toast message is not enaugh in this case ?

Reply:
this it just to not coerce error to Error, actually this throw will never happen